minimesos/src/main/java/com/containersol/minimesos/mesos/ClusterArchitecture.java

```diff
@@ -216,6 +224,7 @@ public Builder withAgent() {
          * All default instance, but with defined resources
          *
          * @param agentResourcesConfig agent resources configuration
+         * @return TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)